types/chrome/index.d.ts

```diff
@@ -7702,14 +7712,27 @@ declare namespace chrome.sessions {
      * Parameter devices: The list of sessions.Device objects for each synced session, sorted in order from device with most recently modified session to device with least recently modified session. tabs.Tab objects are sorted by recency in the windows.Window of the sessions.Session objects.
      */
     export function getDevices(callback: (devices: Device[]) => void): void;
+    /**
+     * Reopens a windows.Window or tabs.Tab.
+     * @param sessionId Optional.
+     * The windows.Window.sessionId, or tabs.Tab.sessionId to restore. If this parameter is not specified, the most recently closed session is restored.
+     * @return The `restore` method provides its result via callback or returned as a `Promise` (MV3 only).
```

Comment:
I'd avoid mentioning the name again. Also this JSDoc is specific to promises so there's no need to mention callbacks. 

```suggestion
     * @return A promise that resolves with the Session data (MV3 only).
```

This also applies to the other method documented here.

Reply:
So, from what I found, out of 242 promise based definitions 212 use the first description of returned value (#53120) while 7 use the second description.
While I personally prefer your succint take I am not sure if we want to ensure consistency with the previous descriptions, update to new or change them on the fly.